session.go

```diff
@@ -366,6 +366,30 @@ func (s *session) send(frame []byte) (open bool) {
 	return
 }
 
+// addPadding adds random sized padding to the byte slice. The size is capped
+// by s.maxPadding and the slice length, whichever is lower. The slice is then
+// zeroed out. It returns the padding size if there's no error.
+func (s *session) addPadding(b []byte) (int, error) {
+	if !s.paddingEnabled {
+		return 0, nil
+	}
+	padding, err := rand.Int(rand.Reader, s.maxPadding)
+	if err != nil {
+		return 0, err
+	}
+	l := int(padding.Int64() + 1) // have at least 1 byte of padding
+	if l > len(b) {
+		l = len(b)
```

Comment:
I'm not sure if it was the case before, but seems we should cap the padding size.

Reply:
Makes sense.